File: src/N81-Backend/AirBnbServerApp/AirBnb.Domain/Entities/Location.cs
using AirBnb.Domain.Common.Models;
using System.Text.Json.Serialization;

namespace AirBnb.Domain.Entities;

public class Location : IEntity
{
    public Guid Id { get; set; }

    public string PlaceName { get; set; } = default!;

    public string BuiltIn { get; set; } = default!;

    public int AwayInKm { get; set; }

    public string Days { get; set; } = default!;

    public int Price { get; set; }

    public decimal Rate { get; set; }

    public string? ImageUrl { get; set; }

    public Guid CategoryId { get; set; }

    [JsonIgnore]
    public virtual LocationCategory? Category { get; set; } = default;
}

File: src/N82.Client/src/modules/models/Location.ts
import type { Guid } from "guid-typescript";

export class Location{
    public id:Guid;
    public placeName:string;
    public builtIn:string;
    public awayInKm:number;
    public days:string;
    public price:number;
    public rate:number;
    public imageUrl:string;
}